gremlin-go/driver/connection_test.go

```diff
@@ -22,7 +121,7 @@ func TestConnection(t *testing.T) {
 
 	t.Run("Test connection.write()", func(t *testing.T) {
```

Comment:
Seems like this test (and one below) now needs to be adjusted?

```suggestion
	t.Run("Test connection.write()", func(t *testing.T) {
		if runIntegration {
			connection, err := createConnection(testHost, testPort, newLogHandler(&defaultLogger{}, Info, language.English))
			assert.Nil(t, err)
			assert.NotNil(t, connection)
			request := makeStringRequest("g.V().count()")
			resultSet, err := connection.write(&request)
			assert.Nil(t, err)
			assert.NotNil(t, resultSet)
			result := resultSet.All()
			assert.Equal(t, false, resultSet.IsEmpty())
			assert.NotNil(t, result)
			assert.Equal(t, 1, len(result), "Count should return only one result")
			intResult, _ := result[0].GetInt()
			assert.Equal(t, 5, intResult, "There should be only 5 people created in Test addV()")
			resultSet.Close()
			err = connection.close()
			assert.Nil(t, err)
		}
	})
```